src/style.css

```diff
@@ -1 +1,84 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  display: block;
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
Hello, Nice PR, here is some tips,
On development, we always use a "css normalizer", you can search for it but one of the main thing is using box-sizing: border-box in all elements. To achieve it, you can just write it on top of your css

```css
*,
::before,
::after {
  box-sizing: border-box; /* 1 */
  border-width: 0; /* 2 */
  border-style: solid; /* 2 */
}
```

Reply:
Thanks @alysonvilela for the review!
I added the "css normalizer" and now I will also do it in the next projects.